live-build/misc/migration-scripts/dx_execute

```diff
@@ -98,12 +98,32 @@ LX_RDS="$LX_RDS_PARENT/root"
 	die "the expected Linux RDS ($LX_RDS) was either not found or has" \
 		"been specified more than once in the bootloader's menu file"
 
+#
+# Re-mount the root dataset and run the migration logic.
+#
+rm -rf /var/delphix/migration
+mkdir -p /var/delphix/migration ||
+	die "failed to create /var/delphix/migration"
+MIGRATION_SCRIPT="/opt/delphix/migration/migrate_config.py"
+LX_CONTAINER="${LX_RDS_PARENT##*/}"
+LX_RDS_MOUNT="/tmp/delphix.$LX_CONTAINER/root"
+mkdir -p "$LX_RDS_MOUNT"
+mount -F zfs -o ignoremountpoint "$LX_RDS" "$LX_RDS_MOUNT" ||
+	die "failed to mount the root Linux dataset"
+"${LX_RDS_MOUNT}${MIGRATION_SCRIPT}" pre-upgrade \
+	>>/var/delphix/migration/log 2>&1 || die "failed to run migration"
+#
+# Create a flag file that notifies the delphix-migration service that
+# post-reboot migration logic should be run.
+#
+touch /var/delphix/migration/perform-migration ||
```

Comment:
Not sure if it would be helpful or not, but do we want to remind the reader in the comment above that our current `/var/delphix/` will be migrated as is to Linux?

Not a big deal, I was just confused for a moment and then remembered your latest changes.